libs/algorithms/tests/unit/algorithms/transform_binary_tests.hpp

```diff
@@ -67,9 +67,9 @@ void test_transform_binary(ExPolicy policy, IteratorTag)
     auto result = hpx::parallel::transform(policy, iterator(std::begin(c1)),
```

Comment:
Please change all uses of `hpx::parallel::transform` to `hpx::transform` or `hpx::ranges::transform` as appropriate.

Reply:
I had let them like this in order to test if they run properely with the changed code. I will change them ASAP though.

Reply:
Ahh, makes sense. Take your time with the changes as you see fit.